packages/now-go/index.ts

```diff
@@ -436,4 +460,111 @@ Learn more: https://vercel.com/docs/runtimes#official-runtimes/go
   };
 }
 
-export { shouldServe };
+function isPortInfo(v: any): v is PortInfo {
+  return v && typeof v.port === 'number';
+}
+
+function isReadable(v: any): v is Readable {
+  return v && v.readable === true;
+}
+
+async function copyEntrypoint(entrypoint: string, dest: string): Promise<void> {
+  const data = await readFile(entrypoint, 'utf8');
+
+  // Modify package to `package main`
+  const patched = data.replace(/\bpackage\W+\S+\b/, 'package main');
+
+  await writeFile(join(dest, 'entrypoint.go'), patched);
+}
+
+async function copyDevServer(
+  functionName: string,
+  dest: string
+): Promise<void> {
+  const data = await readFile(join(__dirname, 'dev-server.go'), 'utf8');
+
+  // Populate the handler function name
+  const patched = data.replace('__HANDLER_FUNC_NAME', functionName);
+
+  await writeFile(join(dest, 'vercel-dev-server-main.go'), patched);
+}
+
+export async function startDevServer(
+  opts: StartDevServerOptions
+): Promise<StartDevServerResult> {
+  const { entrypoint, workPath, meta = {} } = opts;
+  const { devCacheDir = join(workPath, '.vercel', 'cache') } = meta;
```

Comment:
```suggestion
  const { devCacheDir = join(workPath, '.now', 'cache') } = meta;
```

The only time this will be empty is on old versions of Now CLI so I think it should fallback to `.now`, right? Like the we do here:

https://github.com/vercel/vercel/blob/93a9e5bed36c2db7bb907035b5b372529272f4d5/packages/now-node/src/index.ts#L487

Reply:
Well, again, this default is really only here to appease TypeScript since `devCacheDir` is an optional property on the `Meta` object. In reality, `startDevServer()` functionality isn't even implemented in CLI until _after_ the `devCacheDir` was implemented, so practically, this default will never be used.